src/brogue/Monsters.c

```diff
@@ -156,8 +142,8 @@ creature *generateMonster(short monsterID, boolean itemPossible, boolean mutatio
 
     initializeGender(monst);
 
-    if (!(monst->info.flags & MONST_INANIMATE) && !monst->status[STATUS_LIFESPAN_REMAINING]) {
```

Comment:
I removed `!monst->status[STATUS_LIFESPAN_REMAINING]` because, from my reading of the code, at no point within `generateMonster` does this status get set to a non-zero value, so the check doesn't seem to have any purpose.

I added the condition `(monst->info.abilityFlags & MA_ENTER_SUMMONS)`, which does make phylacteries and phoenix eggs count towards auto-ID where they previously didn't. 

Maybe I'm missing something though. Happy to back it out either of these if you want.

Reply:
> I added the condition `(monst->info.abilityFlags & MA_ENTER_SUMMONS)`, which does make phylacteries and phoenix eggs count towards auto-ID where they previously didn't.

I remember why I added them count for auto-ID. It's in the purgatory check. But we could change that condition.

Reply:
Ah ok. No it's no problem, it's just I couldn't see what it was for. I know it's minor but would be good for the change file

Reply:
Ok i'll add a note to the change file that destroying a phylactery or phoenix egg now counts towards auto-id.

Reply:
Note added.